### ISSUE TITLE
Font size does not affect latex previews

### ISSUE THREAD

**Issue by Eloitor** [TobiasZawada/texfrag#29]
I find the previews too small, but increasing the size of the font does not increase the size of the previews.

**Comment by TobiasZawada** [comment]
Thanks for the report. Previews should be scaled in text-scale-mode.

1. Please add your Emacs version 
2. Can you reproduce the effect with a minimal init file and attach this file here? Or maybe, it is already reproducible with an empty init file?

**Comment by Eloitor** [comment]
My emacs version is `28.1` I was in text-scale-mode and the images did not scale with the text.

My minimal init file is the one recomended in melpa
```
(require 'package)
(add-to-list 'package-archives '("melpa" . "https://melpa.org/packages/") t)
;; Comment/uncomment this line to enable MELPA Stable if desired.  See `package-archive-priorities`
;; and `package-pinned-packages`. Most users will not need or want to do this.
;;(add-to-list 'package-archives '("melpa-stable" . "https://stable.melpa.org/packages/") t)
(package-initialize)
```
I opened emacs and installed `markdown-mode` and `texfrag-mode`.
I then opened this markdown file:
```
# Test

$x^2$ A
```

And after zooming in I see this:

![image](https://user-images.githubusercontent.com/17795572/181054408-7b6a1bb6-26c5-45c7-86aa-1d929ed09950.png)

**Comment by TobiasZawada** [comment]
Thanks for the infos. As workaround, you can customize `texfrag-scale` to a factor larger than 1.0.
Please, report whether this works for you.

**Comment by Eloitor** [comment]
The workaround works for me, thanks.

**Comment by TobiasZawada** [comment]
When you scaled up the text with `text-scale-mode`, did you regenerate the previews with <kbd>C-c C-p C-d</kbd>? If you do that, does the font size of the formula adapt to that one of the text?

**Comment by Eloitor** [comment]
Yes, I didn't do `C-c C-p C-d` after scaling the text. Now I did and the font size of the formula adapted fine. I will add a hook to run `texfrag-document` after `tex-scale-mode`. I think this could be added in the README for reference.

**Comment by TobiasZawada** [comment]
Adding `texfrag-document` after each rescaling of the text might become problematic for large documents. Therefore, it is not implemented. But, maybe, one could regenerate only all currently visible fragments.

I have to think about that feature.
But, I am rather busy right now. So, this will take some time.
I keep the ticket open.
